code/go/0chain.net/smartcontract/storagesc/benchmark_rest_tests.go

```diff
@@ -389,6 +389,62 @@ func BenchmarkRestTests(
 				},
 				Endpoint: srh.getAllocBlobberTerms,
 			},
+			{
+				FuncName: "getUserLockedTotal",
```

Comment:
this endpoint is moved to 0box, please remove it

Reply:
This endpoint is still in 0chian. We want complete benchmark coverage, so we should remove the endpoint from 0chian not just the benchmark.

Reply:
https://github.com/0chain/0chain/pull/2442 added PR to remove `getUserLockedTotal` from 0chain.